pkg/conversion/mapi2capi/aws.go

```diff
@@ -514,6 +510,18 @@ func convertKMSKeyToCAPI(kmsKey mapiv1.AWSResourceReference) string {
 	return ""
 }
 
+func convertEBSVolumeSizeToCAPI(bdm *mapiv1.BlockDeviceMappingSpec) int64 {
+	// If bdm.EBS.VolumeSize is not nil, return the VolumeSize
+	if bdm.EBS.VolumeSize != nil {
+		return *bdm.EBS.VolumeSize
+	}
+
+	// Otherwise, return the default VolumeSize (in GB)
+	defaultVolumeSize := int64(120)
```

Comment:
Would appreciate if we could explain where this 120 value came from? The installer?

Reply:
Sorry, I commented on being unsure if this is the right value for the default before refreshing the page so I had not seen this comment :) yes, i'm taking this from the installer, but I didn't go terribly deep into that research yet (like checking if it's ever changed there, etc) so let me do some further research to craft a more thoughtful response

Reply:
Ok cool, if you can link/explain where this came from that would be helpful, and understanding how it's changed over time is also pretty useful. The installer folks may be able to help answer that question

Reply:
I've done some digging and seen that a conversation was had about it in 2021 [here](https://github.com/openshift/openshift-docs/issues/35793#issuecomment-915504739), it seems like 100GB was the default, but 120GB was set to make sure the E2E tests pass reliably. Happy to ask the installer folks about this still if you think it could be necessary? Perhaps I'm being naive, but Is there a huge difference in setting it at 120GB instead of 100GB? :)

Reply:
I'm fairly happy to keep it at 120 if that's what we currently recommend for our CI